budget_proj/requirements.txt

```diff
@@ -11,3 +11,4 @@ requests==2.13.0
 six==1.10.0
 
 gunicorn
+mixer==5.6.6
```

Comment:
Good catch Jim!  I forgot to mention that this was an experiment to see what happens if a dependency is missed from the root-level `requirements.txt`.  There are no immediate consequences, so I'll push a PR to delete that file, and then if things go haywire for some reason down the line, there'll be a specific marker to explain why we wiped that file.